### ISSUE TITLE
[import/newline-after-import] `considerComments` ignores `count` option

### ISSUE THREAD

**Issue by Arkellys** [import-js/eslint-plugin-import#2710]
With the rule `newline-after-import`, when `considerComments` is `true` it only works with 1 space regardless of the option `count`.

For example, with `{ "count": 2, "considerComments": true }`, this is considered valid:

```js
import defaultExport from './foo'

// some comment here.
const FOO = 'BAR'
```

While it should be:

```js
import defaultExport from './foo'


// some comment here.
const FOO = 'BAR'
```

The `--fix` also acts weirdly with comments, if I set `space` to something > 2 it never adds more than two spaces.

**Comment by xsjcTony** [comment]
+1

**Comment by Arkellys** [comment]
Enabling the new option `exactCount` seems to fix this issue. But shouldn't is work without it?

**Comment by Arkellys** [comment]
Hmm, it seems it intrduced a new bug with `require` statements. With the following config:

```js
"import/newline-after-import": ["warn", { count: 2, considerComments: true, exactCount: true }],
```

This is considered valid while it shouldn't since a newline is missing:

```js
const { join } = require("path");

// some comment here.
const FOO = "BAR";
```

This is considererd invalid, while it should be:

```js
const { join } = require("path");


// some comment here.
const FOO = "BAR";

**Comment by Arkellys** [comment]
I deleted my previous posts because I had time to make further tests.

This issue seems to be resolved with v2.29.0, but there still is a problem regarding `require` statement and `considerComments`. I will create a new issue for this.
